go.mod

```diff
@@ -11,6 +11,8 @@ replace (
 	github.com/knative/pkg => github.com/rancher/pkg v0.0.0-20190514055449-b30ab9de040e
 	github.com/matryer/moq => github.com/rancher/moq v0.0.0-20200712062324-13d1f37d2d77
 	github.com/opencontainers/runc => github.com/opencontainers/runc v1.0.1
+
+	github.com/rancher/norman => github.com/rancher/norman v0.0.0-20210709145327-afd06f533ca3
```

Comment:
need to fix, was having issues with go mod tidy that I just wanted to be able to test.